test/networkd-test.py

```diff
@@ -42,6 +42,8 @@
                                    'systemd-networkd']) == 0
 have_dnsmasq = shutil.which('dnsmasq')
 
+RESOLV_CONF = '/run/systemd/resolve/resolv.conf'
+
 
```

Comment:
No, it's not. PEP-8 mandates two empty lines before class declarations, otherwise `pep8` complains. There are two empty lines in the current version as well.

But indeed there was another PEP-8 issue and a typo. Force-pushed the fix.